contracts/mocks/VotesMock.sol

```diff
@@ -21,7 +21,9 @@ abstract contract VotesMock is Votes {
     }
 
     function _mint(address account, uint256 voteId) internal {
-        _balances[account] += 1;
+        unchecked{
+            _balances[account] += 1;
+        }
```

Comment:
we don't care about the gas efficiency of mocks. Lets leave them as they are for the sake of readability.

Reply:
Hello, please with all due respect, I have been away from my office for a while now due to my health issues, please what do I have to do immediately

Reply:
If you can help me I will be very grateful, 
Few months ago I had an accident and I have a memory lost, it took me more often time than not before I could also remember all

Reply:
Mr Amxx